control-plane/tests/test_service_persistence_action_reconciliation_create_tracking_ticket.py

```
@@ -1168,6 +1168,119 @@ def test_service_fail_closes_when_create_tracking_ticket_reconciliation_receipt_
             (downstream_binding["external_receipt_id"],),
         )
 
+    def test_service_fail_closes_when_shuffle_reconciliation_correlation_drifts(
+        self,
+    ) -> None:
+        store, service, approved_payload, delegated_at = (
```

Ruff (0.15.12)

[warning] Unpacked variable `store` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)